docs/reference/ssl/swarm-with-traefik.md

```diff
@@ -0,0 +1,319 @@
+            resources:
+                # limits:   # Enable if you want to limit memory usage
+                #     memory: 200M
+                reservations:
+                    memory: 100M
+            restart_policy:
+                condition: on-failure
+                delay: 5s
+                max_attempts: 20
+                window: 380s
+            placement:
+                constraints:
+                    - 'node.platform.os == linux'
+        secrets:
+            - basic-auth-user
+            - basic-auth-password
+...
+```
+
+### Configure data volumes
+
+Finally, while configuring Traefik, you mounted a volume called `acme`.  You must now define the `acme` volume used for storing Let's Encrypt certificates. We can define an empty volume, meaning data will not persist if you destroy the container. If you destroy the container, the certificates will be regenerated the next time you start Traefik.
+
+Add the following volumes directive on the last line of the file:
+
+```yaml
+...
+volumes:
+    acme:
+```
+
+## Install OpenFaaS
+
+With all of the modifications in place, you can now deploy using the standard deployment scripts. On Mac and Linux: `deploy_stack.sh` and on Windows use `deploy_stack.ps1`. This script will deploy all of the required resources (services, configuration files, networks, and secrets) for your OpenFaaS cluster as they are defined in the `docker-compose.yaml` file.
+
+First, make sure that Docker will execute commands on your Swarm manager node:
+
+```bash
+eval $(docker-machine env <name-of-manager-node>)
+```
+
+Then, run your deployment script. A successful install will look like this:
+
+```bash
+$ ./deploy_stack.sh
+Attempting to create credentials for gateway..
+adadsf809adf098adfajlk12g
+mb1jl213nlkmqfadsaokv68lh
+[Credentials]
+ username: admin
+ password: <randomly-generated-password>
+ echo -n <randomly-generated-password> | faas-cli login --username=admin --password-stdin
+
+Enabling basic authentication for gateway..
+
+Deploying OpenFaaS core services
+Creating network func_functions
+Creating config func_alertmanager_config
+Creating config func_prometheus_config
+Creating config func_prometheus_rules
+Creating service func_alertmanager
+Creating service func_traefik
+Creating service func_gateway
+Creating service func_faas-swarm
+Creating service func_nats
+Creating service func_queue-worker
+Creating service func_prometheus
+```
+
+## Deploy and Invoke a function
+
+In your projects containing OpenFaaS functions, you can now deploy using your domain as the gateway.
+
+### Deploy from the CLI
+```sh
+faas-cli login --gateway https://openfaas.mydomain.com --username admin --password <randomly-generated-password>
+faas-cli deploy --gateway https://openfaas.mydomain.com
+```
+Replace `openfaas.mydomain.com` with your domain as well as adding the username `admin` and secure random password that the deploy script created for you when you deployed OpenFaaS.
+
+### Using the web UI
```

Comment:
Please prefer an active style for headings:

Use vs using
Setup vs setting up
Create a vs creating a